stack/keycloak/migration/02_add-client-pp.yml

```diff
@@ -43,5 +42,17 @@ changes:
   - addRole:
       clientId: ${SSO_CLIENT_ID}
       clientRole: true
-      name: clientrole_testuser
-      description: Role for a testuser
\ No newline at end of file
+      name: AUSBILDUNGSLEITUNG
+      description: Role for örtl. Ausbildungsleitung
+
+  - addRole:
+      clientId: ${SSO_CLIENT_ID}
+      clientRole: true
+      name: NWK
+      description: Role for Nachwuchskräfte
+
+  - addRole:
+      clientId: ${SSO_CLIENT_ID}
+      clientRole: true
+      name: AUSBILDER
+      description: Role for örtl. Ausbilder
```

Comment:
like when using flyway it is not recommended to change already applied files. Users have to reset its keycloak. In my opinion that is not very dev-friendly